src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-cluster/nfs-cluster.component.html

```diff
@@ -0,0 +1,51 @@
+<ng-container *ngIf="orchStatus?.available && clusters$ | async as clusters">
+<cd-table [autoReload]="true"
+          [data]="clusters"
+          columnMode="flex"
+          [columns]="columns"
+          selectionType="single"
+          [hasDetails]="true"
+          (setExpandedRow)="setExpandedRow($event)"
+          (fetchData)="loadData()"
+          (updateSelection)="updateSelection($event)">
+<cd-table-actions class="table-actions"
+                  [selection]="selection"
+                  [permission]="permission"
+                  [tableActions]="tableActions"></cd-table-actions>
+<cd-nfs-cluster-details  *cdTableDetail
+                         [selection]="expandedRow"></cd-nfs-cluster-details>
```

Comment:
i assume this might trigger lint errors? usually something inside an html tag needs to have 2 empty spaces before
```suggestion
  <cd-table-actions class="table-actions"
                  [selection]="selection"
                  [permission]="permission"
                  [tableActions]="tableActions"></cd-table-actions>
  <cd-nfs-cluster-details  *cdTableDetail
                         [selection]="expandedRow"></cd-nfs-cluster-details>
```